java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -2442,7 +2442,10 @@ open class KotlinFileExtractor(
                         }
                     }
                     tw.writeExprsKotlinType(id, type.kotlinResult.id)
-                    unaryopDisp(id)
+                    if (target.name.asString() == "inv" && (c.type.isByte() || c.type.isShort()))
```

Comment:
Does `isFunction` work on extensions? Would `isFunction(target, "kotlin.experimental", "Byte", "inv")` match on this target? This way we would also check the package and not just the function name.

Reply:
OK, done (though it checks for the package of Byte / Short, not the package where the ext method is declared)